src/models/categories.ts

```diff
@@ -1,5 +1,6 @@
 import { Model, DataTypes, CreationOptional } from 'sequelize';
 import { connection } from '../database';
+import Product from './products';
```

Comment:
Why do we need this import? Are we using it? If you are using it for the associations remember that you should be passing the models from the index file, and not import it in each file.